source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,184 @@
+.. _entity-framework-indexes:
+
+=============================
+Optimize Queries with Indexes
+=============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, index, optimization, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **indexes** in the {+provider-short+}.
+Indexes can improve the efficiency of queries and add additional functionality
+to querying and storing documents.
+
+Without indexes, MongoDB must scan every document in a collection to find the
+documents that match each query. These collection scans are slow and can
+negatively affect the performance of your application. However, if an
+appropriate index exists for a query, MongoDB can use the index to limit the
+documents it must inspect.
+
+To improve query performance, build indexes on fields that appear often in your
+application's queries and operations that return sorted results. Each index that
+you add consumes disk space and memory when active, so we recommend that you
+track index memory and disk usage for capacity planning. In addition, when a
+write operation updates an indexed field, MongoDB updates the related index.
```

Comment:
S: this last sentence doesn't really connect to the prev. Consider moving into a note, or changing the transition from "in addition" to something more connected such as "Each write operation that changes an indexed field
triggers an update to the index. This process consumes more memory, so ...."

Reply:
Agreed, I took this from the server page, but reading it again it didn't seem necessary for this page so I just deleted it